vmware/container-service-extension
# [VCDA-2153] cse run to load the schema based on rde_in_use

Signed-off-by: Aniruddha Shamasundar <aniruddha.9794@gmail.com>

To help us process your pull request efficiently, please include: 

- Introduce a map for rde version to def keys
- deprecate methods which doesn't use rde-version to get entities

Testing done:
* check if cluster create is working fine with VCD 10.2
* Check if cluster list is working from CLI

- (Required) Detailed description of changes include tests and
  documentation.  If the pull request contains multiple commits with 
  detailed messages, refer to those instead

@sahithi @sakthisunda @sahithi @rocknes @ltimothy7


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/vmware/container-service-extension/925)
